scripts/zkai_claim_audit_comparison_artifacts_gate.py

```
@@ -0,0 +1,941 @@
+    matched_workload = require_bool(row["matched_workload"], f"{row_id} matched workload")
+    native_equivalent = require_bool(row["native_proof_equivalent"], f"{row_id} native equivalent")
+    proof_size_comparable = require_bool(row["proof_size_comparable"], f"{row_id} proof-size comparable")
+    require_str(row["source_artifact"], f"{row_id} source artifact")
+    require_str(row["system"], f"{row_id} system")
+    require_str(row["workload"], f"{row_id} workload")
+    require_str(row["primary_metric"], f"{row_id} primary metric")
+    if claim_boundary.lower() in {"matched", "comparable"}:
+        raise ClaimAuditError(f"{row_id} claim boundary too weak")
+    if is_external_source_status(source_status) and locally_reproduced:
+        raise ClaimAuditError(f"{row_id} external source marked local")
+    local_like_status = source_status.startswith("local") or source_status.startswith("GO") or source_status == "LOCAL_COMPONENT_FRONTIER"
```

Ruff (0.15.12)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)